### DISCUSSION TITLE
Image is look like transparent when view img.png

### DISCUSSION THREAD

**Discussion by emourdavid** [d2phap/ImageGlass#1803] [General] [answered]
When I view img.png on ImageGlass, the frame will look like transparent so the img will be more blur. Do you how to fix this? 

![image](https://github.com/d2phap/ImageGlass/assets/139842563/55515713-379e-4fc2-a4d9-bd681a6a50da)

**Comment by d2phap** [comment, marked as answer]
Pls refer to  https://github.com/d2phap/ImageGlass/issues/1744#issuecomment-1836201850

**Reply by emourdavid** [reply]
Thank you
